.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig http://EditorConfig.org
+
+# Project Root
+root = true
+
+# Default Code Style
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = tab
```

Comment:
We don't want tabs *anywhere* on the CKAN code base :) docs, requirements files, Dockerfiles, etc So I'm +1 on setting `indent_style = space` globally